libs/core/src/lib/avatar-group/avatar-group.component.html

```diff
@@ -0,0 +1,6 @@
+<div #avatarGroupContainer
```

Comment:
Shouldn't we force to render items in one row only by `overflow: hidden` and `whitespace: nowrap`?
Sometimes I can see that items float on the second row...

![image](https://user-images.githubusercontent.com/3263744/110627157-4a8b5200-81aa-11eb-984e-7ac9bca8d913.png)